### PR TITLE
SW-462_allow_routerinterfaces_in_export

### PR DESCRIPTION
**Pull request by dblaichinger**

### CODE REVIEW

File: src/Controller/ExportController.php [hunk 1]
```diff
@@ -35,6 +35,7 @@
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Routing\RouterInterface;
```
Comment (TheKeymaster) [inline src/Controller/ExportController.php:38]: I guess we can remove the old `Router` import?

Comment (dblaichinger) [reply src/Controller/ExportController.php:36]: `Route` and `RouteScope` was unused according to IDE, removed it

File: src/Controller/ExportController.php [hunk 2]
```diff
@@ -25,7 +25,6 @@
 use Shopware\Core\Framework\DataAbstractionLayer\Search\EntitySearchResult;
 use Shopware\Core\Framework\DataAbstractionLayer\Search\Filter\EqualsFilter;
 use Shopware\Core\Framework\DataAbstractionLayer\Search\IdSearchResult;
-use Shopware\Core\Framework\Routing\Annotation\RouteScope;
```
Comment (TheKeymaster) [inline src/Controller/ExportController.php:28]: This should not be removed, as it used in a Symfony annotation. Please install the Symfony Plugin for your IDE. With it this won't count as "not imported". Also the tests break due to this :).

Comment (dblaichinger) [reply src/Controller/ExportController.php]: After Symfony plugin was installed, nothing seems unused according to IDE so I just reverted the last commit.

**Pull request merged**